test/Makefile

```diff
@@ -3,6 +3,8 @@
 #   make [all]               - makes everything.
 #   make TARGET              - makes the given target.
 #   make run                 - makes everything and runs all the tests.
+#   make run_tests           - run all tests
+#   make run_%               - replace % with specific test file (exclude _test.cpp) to run it, eg make run_IRSend runs tests within IRSend_test.cpp, also makes everything first
```

Comment:
Minor Style nitpick:
Please limit lines to 80 chars (max) where practical. i.e. Please split this into multiple lines.
See https://github.com/crankyoldgit/IRremoteESP8266/wiki/Library-Maintainers-Guide#general